wiki/content/graphql/lambda/overview.md

```diff
@@ -0,0 +1,195 @@
++++
+title = "Overview"
+weight = 1
+[menu.main]
+    parent = "lambda"
+    identifier = "lambda-resolvers-overview"
++++
+
+Lambda provides a way to write your custom logic in JavaScript, integrate it with your GraphQL schema, and execute it using the GraphQL API in a few easy steps:
+
+- Setup a Dgraph cluster with a working lambda server (not required for [Slash GraphQL](https://dgraph.io/slash-graphql) users)
+- Declare lambda queries, mutations, and fields in your GraphQL schema as needed
+- Define lambda resolvers for them in a JavaScript file
+
+This also simplifies the job of UI developers, allowing them to work in JavaScript, without knowing any other backend language.
```

Comment:
... simplifies the job of developers, as they can build a complex backend that is rich with business logic, without setting up multiple different services. Also, you can build your backend in Javascript, which means you can build both your frontend and backend in the same language.

Reply:
Done